packages/ckeditor5-ui/src/dropdown/utils.ts

```diff
@@ -677,7 +678,7 @@ function bindViewCollectionItemsToDefinitions(
 			groupView.items.delegate( 'execute' ).to( dropdownView );
 
 			return groupView;
-		} else if ( def.type === 'button' || def.type === 'switchbutton' ) {
+		} else if ( def.type === 'button' || def.type === 'switchbutton' || def.type === 'labelledbutton' ) {
```

Comment:
Can't we just make `def.label` a general property for when `def.type === 'button'` (in short: add it to `ListDropdownButtonDefinition`) and avoid adding additional type here? It's not that different, it deserves a new type IMO.